dyn/tm/services/dsf.py

```diff
@@ -4142,7 +4142,7 @@ def nodes(self, nodes):
                     _nodeList.append(node)
         elif isinstance(nodes, dict):
             _nodeList.append(nodes)
-        elif isinstance(nodes, DSFNode) or type(node).__name__ == 'Node':
+        elif isinstance(nodes, DSFNode) or type(nodes).__name__ == 'Node':
```

Comment:
using `type` for comparisons is generally frowned upon, could you not just change this line to this:

```python
elif isinstance(nodes, (DSFNode, Node)):
```